src/middlewared/middlewared/plugins/network.py

```diff
@@ -2735,40 +2735,26 @@ async def configure_http_proxy(middleware, *args, **kwargs):
 
 
 async def attach_interface(middleware, iface):
-    if not iface:
-        return
-
-    # We dont handle the following interfaces in middlewared
-    if iface.startswith(('epair', 'tun', 'tap', 'veth', 'kube-bridge')):
-        return
-
-    iface = await middleware.call('interface.query', [('name', '=', iface)])
-    if not iface:
-        return
-
-    iface = iface[0]
-    # We only want to sync physical interfaces that are hot-plugged,
-    # not cloned interfaces with might be a race condition with original devd.
-    # See #33294 as an example.
-    if iface['state']['cloned']:
-        return
-
-    if await middleware.call('interface.sync_interface', iface['name']):
-        await middleware.call('interface.run_dhcp', iface['name'], False)
+    if await middleware.call('interface.sync_interface', iface[0]['name']):
+        await middleware.call('interface.run_dhcp', iface[0]['name'], False)
 
 
-async def devd_ifnet_hook(middleware, data):
-    if data.get('system') != 'IFNET' or data.get('type') != 'ATTACH':
+async def udevd_ifnet_hook(middleware, data):
+    """
+    This hook is called on udevd interface type events. More specifically
+    it should only be called on physical interfaces that are added.
+    """
+    if data.get('SUBSYSTEM', '') != 'net' and data.get('ACTION', '') != 'add':
         return
 
-    await attach_interface(middleware, data.get('subsystem'))
-
-
-async def udevd_ifnet_hook(middleware, data):
-    if data.get('SUBSYSTEM') != 'net' and data.get('ACTION') != 'add':
+    int_name = data.get('INTERFACE', None)
```

Comment:
It would be `None` already if `INTERFACE` is not present

Reply:
Explicitly specifying the default value is proper.